cli/cook/subcommands/cat.py

```diff
@@ -0,0 +1,43 @@
+from functools import partial
+
+from cook.mesos import read_file
+from cook.querying import parse_entity_refs, query_unique_and_run
+from cook.util import guard_no_cluster
+
+
+def cat_for_instance(instance, sandbox_dir, path):
+    """Outputs the contents of the Mesos sandbox path for the given instance."""
+    read = partial(read_file, instance=instance, sandbox_dir=sandbox_dir, path=path)
+    offset = 0
+    chunk_size = 4096
+    data = read(offset=offset, length=chunk_size)['data']
+    while len(data) > 0:
+        print(data, end='')
+        offset = offset + len(data)
+        data = read(offset=offset, length=chunk_size)['data']
+
+
+def cat(clusters, args, _):
+    """Outputs the contents of the corresponding Mesos sandbox path by job or instance uuid."""
+    guard_no_cluster(clusters)
+    entity_refs, clusters_of_interest = parse_entity_refs(clusters, args.get('uuid'))
+    paths = args.get('path')
+
+    if len(entity_refs) > 1:
+        # argparse should prevent this, but we'll be defensive anyway
+        raise Exception(f'You can only provide a single uuid.')
```

Comment:
I think it's good practice to avoid using "you" when reporting errors. Something less direct like *"Only a single UUID is supported" might be better. However, I think this error would be better handled by `argparse`.

Also, we should replace "UUID" with "entity refs" or something like that. More discussion on this below.

Reply:
I have the comment above the line to attempt to make it clear that `argparse` should prevent this from ever happening

Reply:
Ah, yes, I should have noticed the comment. It sounds like this check fits better semantically as an assertion than as an explicit test+raise. E.g.:
```python
# argparse should enforce this
assert len(entity_refs) == 1, "Must provide a single job or instance reference"
```

Reply:
Done